tools/topology/topology1/sof/pipe-codec-adapter-capture.m4

```diff
@@ -30,7 +30,8 @@ include(`bytecontrol.m4')
 #    - [16:31]: reserved 0s
 # - (optional) 12+ bytes codec_param: codec TLV parameters container, for more details please refer
 #                                     struct codec_param under audio/codec_adapter/codec/generic.h
-#    - [0:3]: param ID
+#    - [0:1]: param ID
+#    - [2:3]: codec ID (when supporting multiple codecs, 0 otherwise)
```

Comment:
Just thinking aloud - this looks like the ALSA TLV data descriptor where we have type & size followed by data. What's the reason behind using the bitfields instead of ALSA TLV ? (I suspect for SOF IPC ABI non breakage).

Reply:
@lgirdwood yes, this could be one reason. I don't know the inner details as @mrajwa  (i think) initially added this. 

i have now looked at the documentation:

```

 * SectionControlBytes."name" {
 *»     tlv "tld_data"» »       »       # optional TLV data
 *»     data "pdata for mixer1"»»       # optional private data                                                                                                                                            
 * }
```

and it looks like both tlv, data fit our needs. I don't know exactly how tlv could have made our life easier tough, as the current implementation is clean enough.

Reply:
ok, just checking.